maat-court-data-api/src/main/java/gov/uk/courtdata/billing/repository/ApplicantCclfResetRepository.java

```diff
@@ -0,0 +1,28 @@
+package gov.uk.courtdata.billing.repository;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.Collections;
+
+@Repository
+@RequiredArgsConstructor
+public class ApplicantCclfResetRepository {
+
+    private static final String RESET_CCLF_SQL = """
+        UPDATE APPLICANTS
+        SET SEND_TO_CCLF = NULL,
+            DATE_MODIFIED = SYSDATE,
+            USER_MODIFIED = USER
+        WHERE ID IN (SELECT APPL_ID FROM MAAT_REFS_TO_EXTRACT)
+        """;
+
+    private final NamedParameterJdbcTemplate jdbcTemplate;
+
+    @Transactional
+    public void resetApplicantCclfFlag() {
+        jdbcTemplate.update(RESET_CCLF_SQL, Collections.emptyMap());
```

Comment:
For the update statement as it currently stands, I don't think it's necessary to use the `NamedParameterJdbcTemplate`, especially as we're passing it zero parameters. Potentially for the revised update statement as per my comments above, but it might also be best to stick with how we are doing this sort of update in a few other places throughout the codebase without the use of this template.